utils/misc.mli

```diff
@@ -165,15 +157,6 @@ module Stdlib : sig
         elements, except for possibly the last list, which may have fewer.
         [chunks_of] raises if [n <= 0]. *)
 
-||||||| 121bedcfd2
-=======
-    val chunks_of : int -> 'a t -> 'a t t
```

Comment:
Why are we dropping this part?

Reply:
It's a duplicate (expand the diff just above).